pysrc/scripts/evaluation.py

```diff
@@ -116,24 +214,28 @@ def parse_args() -> argparse.Namespace:
 
 def run_evaluation(config: GlobalConfiguration, args: argparse.Namespace) -> Optional[str]:
     experiments: MetanomeRunBatch = load_experiment_information(json_file=args.file)
-    csv_path = create_evaluation_csv(experiments, config.output_file, config)
+    
+    # The file-names of the evaluations should depend on the source file timestamp, not the current timestamp!
+    output_file = args.file.rsplit('/',1)[-1].rsplit('.', 1)[0]
+    
+    csv_path = create_evaluation_csv(experiments, output_file, config.output_folder)
     if config.create_plots:
-        plot_path = make_plots(config.output_file, config.plot_folder, config)
+        plot_paths = make_plots(output_file, config.plot_folder, config)
     match args.return_path:
         case 'csv':
             return csv_path
         case 'plot':
-            return plot_path
+            return ', '.join(plot_paths)
```

Comment:
Not ideal, but I don't know a better solution for now that would still be human-readable. Originally, the idea was that the output from one step can be passed to the next step. As `, ` is a valid part of a path, we cannot split on it. The computer-readable solution would be separation by `\0` which is not ideal for humans. But as we currently don't have a next stage, this should be fine.